src/main.ts

```diff
@@ -4,18 +4,20 @@ import * as installer from './installer';
 import * as semver from 'semver';
 import path from 'path';
 import {restoreCache} from './cache-restore';
-import {isGhes, isCacheFeatureAvailable} from './cache-utils';
+import {isCacheFeatureAvailable} from './cache-utils';
 import cp from 'child_process';
 import fs from 'fs';
 import os from 'os';
+import {IToolRelease} from '@actions/tool-cache';
+import {StableReleaseAlias} from './utils';
 
 export async function run() {
   try {
     //
     // versionSpec is optional.  If supplied, install / use from the tool cache
     // If not supplied then problem matchers will still be setup.  Useful for self-hosted.
     //
-    const versionSpec = resolveVersionInput();
+    let versionSpec = resolveVersionInput();
```

Comment:
When you move the logic down that Dmitry suggested to installer.ts, you will no longer need this variable to be defined as let, you will be able to use const again.